desktop/renderer/remote-manager.js

```diff
@@ -228,8 +225,7 @@ export default connect(
     trackerStopTimer,
     trackerOnClose,
```

Comment:
i think we had some better idea of cleaning it up that was larger? i dunno. i made a ticket